pep-0690.rst

```diff
@@ -312,57 +325,54 @@ import object. When ``modb.foo`` is later referenced, it will also try to
 Python, and at this point will replace the lazy import object at
 ``modb.__dict__["foo"]`` with the actual module ``foo``.
 
-There is one case in which a lazy import object can escape one dictionary (but
-only into another dictionary) without being resolved. To preserve the
-performance of bulk-copy operations like ``dict.update()`` and ``dict.copy()``,
-they do not check for or resolve lazy import objects. However, if the source
-dict has a ``*_LAZY`` lookup kind set that indicates it might contain lazy
-objects, that lookup kind will be passed on to the updated/copied dictionary.
-This still ensures that the lazy import object can't escape into Python code
-without being resolved.
-
-Other "bulk" dictionary lookup methods (such as ``dict.items()``,
-``dict.values()``, etc) will resolve all lazy import objects in the dictionary.
-Since it is uncommon for any of these to be used on a module namespace
-dictionary, the priority here is simplicity of implementation and minimizing the
-overhead on normal non-lazy dictionaries (just one check to see if the
-dictionary has a ``*_LAZY`` lookup kind).
+There are two cases in which a lazy import object can "escape" one dictionary:
```

Comment:
Well, technically it could escape any dictionary containing lazy objects. It's not necessary it is a modules dictionary. For example, a dictionary built using `globals().copy()`. I'm not so sure what wording is best here for that reason.

Reply:
Maybe just "escape a dictionary"?

Reply:
Sounds good!